src/fmain.pas

```diff
@@ -6774,34 +6892,36 @@ procedure Tfrmain.milistPropertiesClick(Sender: TObject);
           for i:=0 to frmain.lvMain.Items.Count-1 do
           begin
             //// Change only the normal download type and with pause, complete status
-            if (frmain.lvMain.Items[i].SubItems[columntype] = '0') and ((frmain.lvMain.Items[i].SubItems[columnstatus]='0') or (frmain.lvMain.Items[i].SubItems[columnstatus]='3')) and frmain.lvMain.Items[i].Selected then
+            itm := frmain.lvMain.Items[i];
```

Comment:
this is how I would do it in C++ : defining a reference to an indexed variable, and then use this reference instead of repeating the same indexed variable again and again